Mastodon/Scene/Root/MainTab/MainTabBarController.swift

```diff
@@ -359,17 +358,22 @@ extension MainTabBarController {
         guard let tab = touchedTab(by: sender) else { return }
 
         switch tab {
-        case .me:
-            guard let authContext = authContext else { return }
+        // todo: reconsider the "double tap to change account" feature -> https://github.com/mastodon/mastodon-ios/issues/628
+//        case .me:
+//            guard let authContext = authContext else { return }
+//            assert(Thread.isMainThread)
+//
+//            guard let nextAccount = context.nextAccount(in: authContext) else { return }
+//            
+//            Task { @MainActor in
+//                let isActive = try await context.authenticationService.activeMastodonUser(domain: nextAccount.domain, userID: nextAccount.userID)
+//                guard isActive else { return }
+//                self.coordinator.setup()
+//            }
```

Comment:
Good idea to comment this out, what do you think about removing this? We could add it as a comment to #628 so that we don't loose it, but clean up the code a bit?

Reply:
sure thing! Don't know the reason why we kept the todo so thought of not completely removing it. 

Now that you are okay with that. I will go ahead and do some clean up